src/core/client/admin/components/BanModal.css

```diff
@@ -15,3 +15,34 @@ $ban-modal-text: var(--palette-text-500);
 .sitesToggle {
   margin-bottom: var(--spacing-3);
 }
+
+.domainBanHeader {
+  margin-bottom: var(--spacing-2);
+
+}
+
+
+.form {
+  gap: var(--spacing-3);
+  margin-bottom: var(--spacing-3);
+}
+
+.banFromHeader {
+  margin-bottom: var(--spacing-2);
+  margin-top: 0;
+  font-family: var(--font-family-primary);
```

Comment:
Is it necessary to set `font-family` here? Looks like it's already set by the `Label` styles.

Reply:
I'll need to refresh my memory as to why on monday, but without this, the header styles break when opening the Ban Modal from the moderation queue. There might be a better way to address that though.

Reply:
aaaaand after fixing the bug preventing me from reproducing this, I cannot reproduce. So, removed!
✅